src/_actions/WorkspaceActions.js

```diff
@@ -55,3 +55,11 @@ export const delFromWatchlist = symbol => {
         dispatch(workspaceUnfavorAsset(symbol));
     };
 };
+
+export const selectAssetSymbolForTrade = (newSymbol, oldSymbol) => {
+    return dispatch => {
+        LiveData.api.unsubscribeFromTick(oldSymbol);
+        LiveData.api.subscribeToTick(newSymbol);
```

Comment:
Yeah, we'll need to abstract it a bit, but haven't read this thread. Good find ;)